src/server/package-lock.json

```diff
@@ -16,18 +16,22 @@
                 "express": "^4.18.2",
                 "express-session": "^1.17.3",
                 "jsonwebtoken": "^9.0.1",
+<<<<<<< HEAD
```

Comment:
Hii @gbowne1 
Could you know why this package-lock.json is changing with developers...
I seen this in other projects aswell
Wheather this impact on the project?

Reply:
Yeah, it just means the incoming changes need to be accepted on your end. Do you see a line where it says incoming changes?  It happens.  I don't know the full reason behind why but it should not have any impact.  

Thank you for your contribution to CodeBooker @ratishjain12 and for the opportunity to review and merge your PR.